src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Scanner;
+import duke.task.*;
+import duke.exception.*;
```

Comment:
Import classes explicitly instead of using Wildcard (*), to comply with coding standards.